deploy/examples/import-external-cluster.sh

```diff
@@ -152,6 +173,54 @@ function importRGWAdminOpsUser() {
   --from-literal=secretKey="$RGW_ADMIN_OPS_USER_SECRET_KEY"
 }
 
+function createRBDStorageClass() {
+cat <<eof | kubectl create -f -
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: $RBD_STORAGE_CLASS_NAME
+provisioner: $RBD_PROVISIONER
+parameters:
+  clusterID: $NAMESPACE
+  pool: $RBD_POOL_NAME
```

Comment:
what about `dataPool` to support EC pools?

Reply:
Should I keep it blank if not exist,
(dataPool: $RBD_METADATA_EC_POOL_NAME)
LIke if exit,
dataPool: abc
Not exist or provided,
dataPool: 

Or this shouldn't be kept blank?

Reply:
the empty key should not be present. if it's black don't add key:value.

Reply:
So for that I cannot do if-else added a separate function , please check that